src/app/modules/rup/components/ejecucion/buscador.component.ts

```diff
@@ -414,6 +416,55 @@ export class BuscadorComponent implements OnInit, OnChanges {
      *
      * @param {any} concepto Concepto SNOMED
      */
+    public verificarProfesional(concepto, index) {
+        let permiso = false;
+        if (concepto.conceptId === '16076005') {// receta
```

Comment:
Aldito lo veo muy bien, te voy dejando detalles a mejorar que noté.
Acá veo que está perfecto el control por concepto, pero también controlar que no sea una solicitud

Reply:
![image](https://github.com/user-attachments/assets/c00bea3c-91ae-4374-aa7f-5463abb94edc)
agregado! gracias por los comentarios cele!